WebApplication/Backend/Repositorys/FeedbackRepository.cs

```diff
@@ -33,59 +33,24 @@ public FeedbackRepository()
         ///<exception>
         ///if any exception occurs method will return null
         ///</exception>
-        internal List<Feedback> GetAllFeedbacks()
+        internal List<Feedback> GetFeedbacks(String sqlDml)
```

Comment:
Just make this a private method, and make three public methods: GetAllFeedbacks, GetApprovedFeedbacks, GetDisapprovedFeedbacks and set the sql query string in those methods, then call this one from each of them.